saml-core/src/main/java/org/keycloak/saml/processing/core/saml/v2/util/XMLTimeUtil.java

```diff
@@ -102,15 +102,19 @@ public static XMLGregorianCalendar getIssueInstant(String timezone) {
         GregorianCalendar gc = new GregorianCalendar(tz);
         XMLGregorianCalendar xgc = dtf.newXMLGregorianCalendar(gc);
 
+        Long offsetMilis = TimeUnit.MILLISECONDS.convert(Time.getOffset(), TimeUnit.SECONDS);
+        if (offsetMilis != 0) {
+            logger.debug(XMLTimeUtil.class.getName() + " timeOffset: " + offsetMilis);
```

Comment:
Please use `logger.debugf` instead (dtto below)

Reply:
There is no such method in org.keycloak.saml.common.PicketLinkLogger. We would either add the method there or leave it as it is. I'd vote for latter option.

Reply:
As discussed offline, `if (logger.isDebugEnabled())` is used